dev-libs/openssl/openssl-1.0.2k-r1.ebuild

```diff
@@ -242,6 +237,12 @@ multilib_src_install_all() {
 	dodir /etc/sandbox.d #254521
 	echo 'SANDBOX_PREDICT="/dev/crypto"' > "${ED}"/etc/sandbox.d/10openssl
 
-	diropts -m0700
-	keepdir ${SSL_CNF_DIR}/private
+	# Don't keep the sample CA files and their ilk in /etc.
```

Comment:
That phrase refers to the other odd scripts installed in `/etc/ssl/misc`, like the ones that just print out fields of a certificate.  This PR is following an offline discussion with @marineam about distributing the CA demo scripts, but I saw no reason to keep any of the other stuff in there.